datasets/mmchat/README.md

```diff
@@ -0,0 +1,274 @@
+---
+annotations_creators:
+- other
+language_creators:
+- other
+languages:
+- zh
+licenses:
+- mit
```

Comment:
Same comment as in https://github.com/huggingface/datasets/pull/4431

> The data has been crawled from weibo, therefore I don't think this is license is right.
I don't know much about Weibo, but if I recall correctly they claim the copyright of all the content on the platform, and if this is the case you are not allowed to sublicense the data unless you have an agreement with Weibo.
> 
> Maybe you can set this tag to "other-weibo" instead of "mit", and in the Licensing Information section you can redirect users to the terms of service of Weibo (and ideally summarize what people are allowed to do with this data) ?